.github/workflows/publish-dockerhub.yml

```diff
@@ -0,0 +1,40 @@
+name: Publish Images to Dockerhub
+on:
+  release:
+    types:
+      - created
+env:
+  DOCKER_HUB: docker.io
+  DOCKER_USER: ${{ secrets.DOCKER_IO_USER }}
+jobs:
+  publish-pyth-price-service:
+    name: Publish Pyth price service
+    if: ${{ startsWith(github.ref, 'refs/tags/pyth-price-service-v') }}
```

Comment:
If we are conditioning this to a tag prefix, I think it might be easier to just use that as the workflow triggering condition, like we do [here](https://github.com/pyth-network/publisher-exporter/blob/main/.github/workflows/ci.yaml#L6-L7).

Reply:
Oh I didn't know this is possible, I'll change it this way.